### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by CouchPotato302** [3kh0/ext-remover#379] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Before I start, I just want to say that I've been using your repositories since last year when you had created a few games, thanks for your hard work.

I had been informed of Ingot by a friend and I had used it for a couple of days before updating my Chromebook (v107), I wasn't aware that the update would make Ingot unusable, is there a way I can revert the update to be able to use Ingot again? Maybe you could find another glitch/bookmarklet to disable the extension? I've tried using some of your glitches, but none of them worked. The only one that was close to working was:
`javascript:let shim = false;var ids = prompt("extension ids (comma separated)").split(",");setInterval(()=>{ids.forEach((id)=> opener.chrome.developerPrivate.updateExtensionConfiguration({extensionId: id, fileAccess: shim}));shim = !shim;}, 250);`, the only thing that it did, was repeatedly turn my extension on and off again. I think my school/my Chromebook made it so that if the extension is disabled, it would enable it again.

**Comment by obkduishinijf** [comment, marked as answer]
only way to make ingot work again is to downgrade your chromebook, which could get you banned from all devices, and get it on your permanent record, absolutely ruining your chances of getting into a good school.

**Reply by CouchPotato302** [reply]
> only way to make ingot work again is to downgrade your chromebook, which could get you banned from all devices, and get it on your permanent record, absolutely ruining your chances of getting into a good school.

I'm not sure there's a way for admins to check if I downgraded my device. I also don't think that downgrading my device is that suspicious and if it's my first offence, they would put it on my record.

**Reply by Calamitou** [reply]
my friend downgraded his chromebook and turned off wifi when it started back up again and he hasnt been caught for over 2 months

**Comment by skoopy-star** [comment]
im pretty sure that, that javascript is supposed to do that, it does that for me and everything is unblocked and dyknow doesnt work

**Reply by skoopy-star** [reply]
also dont close the page you use the bookmarklet on

**Reply by Aka-but-nice** [reply]
Yea, a lot of people seem to do that, the science behind this is that, once you click view in web store and spam esc it turns the url into about:blank and because of the Same Policy Orgin, the about:blank page has the same features as the extension page so when you run the bookmarklet and put the id of the extension it acts like chrome://extensions is telling the extension to refresh multiple times and only chrome://extensions, does that but once you close that tab it is as chrome://extensions told the extension to stop refreshing
